dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -1988,6 +1988,7 @@ public void testFindAndDeleteReturnsObjects() {
      * OrderBy annotation.
      */
     @Test
+    @SkipIfSysProp(DB_Oracle) //TODO Eclipse link SQL Generation bug on Oracle: https://github.com/OpenLiberty/open-liberty/issues/28545
```

Comment:
28545 was recently fixed. We will need to replace 28545 with a new issue number for different EclipseLink bug, once opened...